### PR TITLE
Update security faq 

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 59ae68cc098e24af7970626d9121b8037457ad84

https://app.netlify.com/sites/preliminary-istio/deploys/5b2166963672df13b1eb3d1a

File: content/help/faq/security/istio-to-not-istio.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ---
-title: Can a service with Istio Auth enabled communicate with a service without Istio?
+title: Can a service with Istio mutual TLS (mTLS) enabled talk to a service without Istio,
+and what about the other way around?
 weight: 20
 ---
-This is not supported currently, but will be in the near future.
+Starting from 0.8 release, a service with Istio mTLS enabled can talk to a service without Istio. Mutual TLS is enabled via [authentication policy]({{home}}/docs/concepts/security/authn-policy.html) and this only specifies the service behavior as a server, not client, which means a mTLS enabled service will still send http traffic (not mTLS) to others unless you explicitly specify it with [destination rule]({{home}}/docs/reference/config/istio.networking.v1alpha3.html#DestinationRule).
```
Comment (geeknoid) [inline content/help/faq/security/istio-to-not-istio.md]: Replace uses of mTLS with "mutual TLS", replace "cert" with "certificate"

File: content/help/faq/security/istio-to-not-istio.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ---
-title: Can a service with Istio Auth enabled communicate with a service without Istio?
+title: Can a service with Istio mutual TLS (mTLS) enabled talk to a service without Istio,
```
Comment (geeknoid) [inline content/help/faq/security/istio-to-not-istio.md]: Can we have a shorter title? Super-long titles don't look too good on the main FAQ page.

File: content/help/faq/security/istio-to-not-istio.md (hunk 1 above)
Comment (wattli) [reply content/help/faq/security/istio-to-not-istio.md]: Done.

File: content/help/faq/security/istio-to-not-istio.md (hunk 2 above)
Comment (wattli) [reply content/help/faq/security/istio-to-not-istio.md]: Done.

File: content/help/faq/security/istio-to-not-istio.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ---
-title: Can a service with Istio Auth enabled communicate with a service without Istio?
+title: Can Istio mutual TLS enabled services communicate with services without Istio?
 weight: 20
 ---
-This is not supported currently, but will be in the near future.
+Starting from 0.8 release, a service with Istio mutual TLS enabled can talk to a service without Istio. Mutual TLS is enabled via [authentication policy]({{home}}/docs/concepts/security/authn-policy.html) and this only specifies the service behavior as a server, not client, which means a mutual TLS enabled service will still send http traffic (not mutual TLS) to others unless you explicitly specify it with [destination rule]({{home}}/docs/reference/config/istio.networking.v1alpha3.html#DestinationRule).
```
[the remark this answers is not in the document]
Comment (wattli) [reply content/help/faq/security/istio-to-not-istio.md]: done. thanks

**Comment by wattli** [comment]
Thanks @geeknoid !

**Pull request merged**